dts/arm/raspberrypi/rp2040.dtsi

```diff
@@ -47,7 +47,7 @@
 
 	soc {
 		uart0: uart@40034000 {
-			compatible = "raspberrypi,rp2-uart";
+			compatible = "raspberrypi,pico-uart";
```

Comment:
What's going on here? This commit is doing more than adding support for a board. Please split this up a bit more and justify the change to the devicetree here. This is a SoC level change to a compatible; it's not just adding a driver.